static/javascripts/custom/main.js

```diff
@@ -69,50 +69,23 @@ $(function ($) {
     // Text Ticker
```

Comment:
Can we use Vuejs and not include any jquery?

Reply:
All JQuery files come from a template that I have been given to work with. I can try replace content bit by bit to phase out JQuery over time.